core/src/main/resources/hudson/tasks/Maven/help-injectBuildVariables.html

```diff
@@ -0,0 +1,3 @@
+<div>
+  Pass all build variables into maven process in form of java properties. This is seldom needed as Jenkins provides it as environment variables anyway.
```

Comment:
and specific variables can be passed by appending `-Dmyvariable=$SOME_BUILD_VAR` to the maven goals?

(at least I believe this works)